src/rdf.rs

```diff
@@ -1,102 +1,91 @@
-use log::{info, error};
+use log::{error, info};
 
-use oxigraph::io::{GraphFormat};
-use oxigraph::store::{Store, QuadIter, SerializerError};
-use oxigraph::model::*;
+use oxigraph::io::GraphFormat;
 use oxigraph::model::vocab::{rdf, xsd};
+use oxigraph::model::*;
+use oxigraph::store::{QuadIter, SerializerError, Store};
 
 use std::collections::HashMap;
 
-const DCT_FORMAT: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://purl.org/dc/terms/format");
-
-const DCAT_DATASET_CLASS: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dcat#Dataset");        
-
-const DCAT_DISTRIBUTION: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dcat#distribution");        
-
-const DCAT_ACCESS_URL: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dcat#accessURL");        
-
-const DCAT_DOWNLOAD_URL: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dcat#downloadURL");    
-    
-const DQV_QUALITY_MEASUREMENT_CLASS: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dqv#QualityMeasurement");   
-
-const DQV_HAS_QUALITY_MEASUREMENT: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dqv#hasQualityMeasurement");    
-
-const DQV_IS_MEASUREMENT_OF: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dqv#isMeasurementOf");  
-    
-const DQV_COMPUTED_ON: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dqv#computedOn"); 
-
-const DQV_VALUE: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("http://www.w3.org/ns/dqv#value"); 
-
-const DCAT_MQA_ACCESS_URL_STATUS_CODE: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("https://data.norge.no/vocabulary/dcatno-mqa#mqa:accessUrlStatusCode");   
-        
-const DCAT_MQA_DOWNLOAD_URL_STATUS_CODE: NamedNodeRef<'_> =
-    NamedNodeRef::new_unchecked("https://data.norge.no/vocabulary/dcatno-mqa#mqa:downloadUrlStatusCode");
+use crate::vocab::{dcat, dcat_mqa, dcterms, dqv};
 
 // Parse Turtle RDF and load into store
 pub fn parse_turtle(turtle: String) -> Store {
```

Comment:
Bør håndtere feil bedre enn å bare `unwrap`e og printe. (`_` må erstattes med en bestemt error-type)
```suggestion
pub fn parse_turtle(turtle: String) -> Result<Store, _> {
```